helpers/toggleToggleRefinementStatus.ts

```diff
@@ -1,10 +1,10 @@
 declare namespace WebdriverIOAsync {
   interface Browser {
-    changeToggleRefinementStatus(): Promise<boolean>;
+    toggleToggleRefinementStatus(): Promise<boolean>;
```

Comment:
why is this one called toggle?

Reply:
FMPOV it was more clear than `change` (but then we have `toggle` 2 times)

Reply:
`clickToggleRefinement` seems to be appropriate here.